quic/s2n-quic-transport/src/connection/connection_container.rs

```diff
@@ -940,6 +955,15 @@ impl<C: connection::Trait, L: connection::Lock<C>> ConnectionContainer<C, L> {
         );
     }
 
+    /// Iterates over all `Connection`s which are waiting to process ACKs,
+    /// and executes the given function on each `Connection`
+    pub fn iterate_new_ack_list<F>(&mut self, mut func: F)
+    where
+        F: FnMut(&mut C) -> ConnectionContainerIterationResult,
+    {
+        iterate_interruptible!(self, waiting_for_ack, waiting_for_ack_link, func);
```

Comment:
Are we ever going to interrupt this iteration? Or do we just process the list in every batch?

Reply:
Still learning the interest list interface more.. Processing all in a batch makes sense.